src/oauth2/service.ts

```diff
@@ -481,7 +481,7 @@ export async function generateAuthorizationCode(options: GenerateAuthorizationCo
       scope: options.scope?.join(' '),
       browser_session_id: options.browserSessionId,
       created_at: Math.floor(Date.now() / 1000),
-      nonce: options.nonce,
+      nonce: options.nonce ?? null,
```

Comment:
@ansariswag this fixed this bug. knex now errors when passing undefined. Not sure if this is new behavior :S

Reply:
@evert , thanks, let me check out and then try.